models/stg_recurly__billing_info_history.sql

```diff
@@ -15,33 +15,33 @@ fields as (
             staging_columns = get_billing_info_history_columns()
         ) }}
     from
-        base
+        base 
 ),
 final as (
     select
         id as billing_id
-        , _fivetran_synced
-        , created_at
-        , updated_at
-        , account_id
-        , first_name
-        , last_name
-        , company
-        , vat_number
-        , billing_phone
-        , billing_street_1
-        , billing_street_2
-        , billing_city
-        , billing_region
-        , billing_postal_code
-        , billing_country
-        , updated_by_ip
-        , updated_by_country
-        , payment_method_object
-        , payment_method_card_type
-        , valid as is_valid
+        , _fivetran_synced 
+        , created_at 
+        , updated_at 
+        , account_id 
+        , first_name 
+        , last_name 
+        , company 
+        , vat_number 
+        , billing_phone 
+        , billing_street_1 
+        , billing_street_2 
+        , billing_city 
+        , billing_region 
+        , billing_postal_code 
+        , billing_country 
+        , updated_by_ip 
+        , updated_by_country 
+        , payment_method_object 
+        , payment_method_card_type 
+        , valid as is_valid 
```

Comment:
Out style standards indicating we should have trailing commas. Can you update this to reflect our standards.

Reply:
Updated across all models.